modules/openapi-generator/src/main/resources/swift6/Extensions.mustache

```diff
@@ -95,15 +97,6 @@ extension Date: ParameterConvertible {
     func asParameter(codableHelper: CodableHelper) -> any Sendable {
         return codableHelper.dateFormatter.string(from: self)
     }
-}
-
-extension ParameterConvertible where Self: Encodable {
-    func asParameter(codableHelper: CodableHelper) -> any Sendable {
-        guard let data = try? codableHelper.jsonEncoder.encode(self) else {
-            fatalError("Could not encode to json: \(self)")
-        }
-        return data.asParameter(codableHelper: codableHelper)
```

Comment:
I was thinking, would it be wrong to just return the data here?

Reply:
Hi @x-sheep when you have some time, could you please take a look at this? Thanks